doc/reference/bluetooth/hci.txt

```diff
@@ -1014,6 +1015,47 @@ by the controller for the requested Handle_Type and Handle.
 When the Read_Tx_Power_Level command has completed, a Command Complete
 event shall be generated.
 
+Zephyr Read USB Transport Mode Command
+======================================
+
+This command read the supported USB transport modes.
+
++--------------------------+-------+--------------------+--------------------+
+| Command                  | OCF   | Command            | Return             |
+|                          |       | Parameters         | Parameters         |
++--------------------------+-------+--------------------+--------------------+
+| Read_USB_Transport_Mode  | 0x010 |                    | Status             |
+|                          |       |                    | Num_Supported_Modes|
+|                          |       |                    | Supported_Mode[i]  |
++--------------------------+-------+--------------------+--------------------+
+
+	Supported_Mode:                                 Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0x00               | H:2 transport mode                   |
+	| 0x01               | H:4 transport mode                   |
```

Comment:
I think we need a proper definition of these modes. I know H:2 and H:4 are used colloquially amongst Bluetooth veterans, but those are references to ancient Bluetooth spec versions and sections in them that do not exist anymore. Either we should refer to sections in the latest core spec or describe the protocol format in detail if such sections do not exist.

Reply:
How about "USB Mode" and "Serial Mode", H4 is currently used for SPI/UART/RPmsg.

Reply:
Ive added some description to each mode.